homework/list.js

```diff
@@ -52,28 +62,26 @@
       buff.appendChild(li);
     });
     ul.appendChild(buff);
-    _wrapper.appendChild(ul);
+    this._wrapper.appendChild(ul);
   }
 
-  function fetchList(afterFetch) {
-    var xhr = new XMLHttpRequest();
-    xhr.open('GET', 'http://127.0.0.1:8000/demo-list-notes.json', true);
-    xhr.responseType = 'json';
-    xhr.onreadystatechange = function(e) {
+  fetchList(afterFetch) {
```

Comment:
nit: You have successfully "Promisified" it, so you don't need to keep this callback-style argument.